dgl_ptm/regen_version.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
can we add some documentation in this file or in a md file in a docs folder?

Reply:
The current md file in the docs folder is about running on snellius. I think it's better to add some documentation to the CONTRIBUTING.md file.